types/types.cc

```diff
@@ -1063,6 +1063,10 @@ const sstring& abstract_type::cql3_type_name() const {
     return _cql3_type_name;
 }
 
+sstring abstract_type::cql3_type_name_without_frozen() const {
+    return cql3_type_name_impl(*this);
+}
```

Comment:
Do the tests exercise this? Send a frozen type and see it without the annotation in DESCRIBE?

Reply:
@dawmd please answer this comment and resolve if it was done

Reply:
Yes. We test it with a custom type that would be printed as frozen if `cql3_type_name` were used instead.